File: files/100-199/131/CH-131.R
library(tidyverse)
library(readxl)

path = "files/CH-131 Table Transformation.xlsx"
input = read_excel(path, range = "C2:I7")
test  = read_excel(path, range = "K2:N12")

result = bind_rows(input[,1:4], input[,c(1,5:7)]) 

all.equal(result, test)
#> [1] TRUE

File: files/100-199/131/CH-131.py
import pandas as pd

path = "CH-131 Table Transformation.xlsx"
input = pd.read_excel(path, usecols="C:I", skiprows=1, nrows=5).rename(columns=lambda x: x.replace('.1', ''))
test = pd.read_excel(path, usecols="K:N", skiprows=1, nrows=11).rename(columns=lambda x: x.replace('.1', '').replace('.2', ''))

result = pd.concat([input.iloc[:, :4], input.iloc[:, [0, 4, 5, 6]]], axis=0).reset_index(drop=True)

print(result.equals(test))  # True